docs/pages/eas-update/rollouts.mdx

```diff
@@ -2,24 +2,29 @@
 title: Rollouts
 hideTOC: true
```

Comment:
```suggestion
```

Let's remove this since there'll be three sections on this page after publishing this doc.